framework/db/Query.php

```diff
@@ -844,6 +849,119 @@ public function addParams($params)
     }
 
     /**
+     * @var array aliases defined for tables used in this query. `tablename => alias`.
```

Comment:
getAlias is not suited for this case, see docs: https://github.com/yiisoft/yii2/pull/10813/files#diff-586d87573b57677fb773d9ca73cf20b3R893